### PR TITLE
support dataplane traffic validation

### PR DESCRIPTION
**Pull request by raych1**
support multiple swaggers and multiple traffic files validation;
support path parameter transform in live validation and example validation.

### CODE REVIEW

File: lib/liveValidation/liveValidator.ts [hunk 1]
```diff
@@ -850,7 +850,7 @@ export const parseValidationRequest = (
     // Provider would be provider found from the path or Microsoft.Unknown
     providerNamespace = providerNamespace || C.unknownResourceProvider;
     if (providerNamespace === C.unknownResourceProvider) {
-      apiVersion = C.unknownApiVersion;
+      //apiVersion = C.unknownApiVersion;
```
Comment (scbedd) [inline lib/liveValidation/liveValidator.ts:853]: :) I see why you're asking now 👍

File: lib/liveValidation/operationValidator.ts [hunk 2]
```diff
@@ -99,6 +99,7 @@ export const validateSwaggerLiveRequest = async (
   }
 
   const pathParam = extractPathParamValue(info.operationMatch!);
+  transformMapValue(pathParam, operation._pathTransform);
```
Comment (ruowan) [inline lib/liveValidation/operationValidator.ts:102]: what does this do?

File: lib/liveValidation/operationSearcher.ts [hunk 3]
```diff
@@ -154,7 +154,7 @@ export class OperationSearcher {
         undefined,
         requestInfo
       );
-      requestInfo.apiVersion = unknownApiVersion;
+      //requestInfo.apiVersion = unknownApiVersion;
```
Comment (ruowan) [inline lib/liveValidation/operationSearcher.ts:157]: why comment here

File: lib/liveValidation/liveValidator.ts (hunk 1 above)
Comment (raych1) [reply lib/liveValidation/liveValidator.ts:853]: you got it~

File: lib/liveValidation/operationSearcher.ts (hunk 3 above)
Comment (raych1) [reply lib/liveValidation/operationSearcher.ts:157]: use matched api version instead.

File: lib/liveValidation/operationValidator.ts (hunk 2 above)
Comment (raych1) [reply lib/liveValidation/operationValidator.ts:102]: transform path parameters value per types. e.g. 'false' - > false.

**Review by ruowan — APPROVED** [review]

**Pull request merged**